build.ps1

```diff
@@ -7,6 +7,8 @@ $ErrorActionPreference = 'Stop'
 .\dev-env\windows\bin\dadew.ps1 sync
 .\dev-env\windows\bin\dadew.ps1 enable
 
+$ARTIFACT_DIRS = if ("$BUILD_ARTIFACTSTAGINGDIRECTORY") { $BUILD_ARTIFACTSTAGINGDIRECTORY } else { Get-Location }
```

Comment:
if `BUILD_ARTIFACTSTAGINGDIRECTORY` is an env. variable then it should be referenced in powershell as `${env:BUILD_ARTIFACTSTAGINGDIRECTORY}`

Reply:
Thanks, I've added `env:`.